tests/playwright-test/test-modifiers.spec.ts

```diff
@@ -130,6 +132,129 @@ test('test modifiers should work', async ({ runInlineTest }) => {
+      `,
+    });
+
+
+    expect(result.exitCode).toBe(0);
+    expect(result.passed).toBe(2);
+    expect(result.skipped).toBe(4);
+
+    const expectTest = expectTestHelper(result);
+    expectTest('no marker', 'passed', 'expected', '');
+    expectTest('example', 'passed', 'expected', '');
+
+    expectTest('skip wrap', 'skipped', 'skipped', 'skip');
+    expectTest('skip inner', 'skipped', 'skipped', 'skip');
+    expectTest('fixme wrap', 'skipped', 'skipped', 'fixme');
+    expectTest('fixme inner', 'skipped', 'skipped', 'fixme');
+  });
+
+  test('should work alongside top-level modifier', async ({ runInlineTest }) => {
+    test.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/15239' });
+    test.fixme(true, 'Need to determine what we expect behavior here to be.');
+
+    test.fixme();
+    const result = await runInlineTest({
+      'a.test.ts': `
+        const { test } = pwt;
+
+        test.fixme();
+
+        test.describe('suite1', () => {
+          test('no marker', () => {});
+          test.skip('skip wrap', () => {});
+          test('skip inner', () => { test.skip(); });
+          test.fixme('fixme wrap', () => {});
+          test('fixme inner', () => { test.fixme(); });
+        });
+
+        test('example', () => {});
+      `,
+    });
+
+
+    expect(result.exitCode).toBe(0);
+    expect(result.passed).toBe(1);
+    expect(result.skipped).toBe(4);
+
+    const expectTest = expectTestHelper(result);
+    expectTest('skip wrap', 'skipped', 'skipped', 'fixme,skip');
+    expectTest('skip inner', 'skipped', 'skipped', 'fixme,skip');
+    expectTest('fixme wrap', 'skipped', 'skipped', 'fixme,skip');
+    expectTest('fixme inner', 'skipped', 'skipped', 'fixme,skip');
+  });
+
+  test('should work alongside top-level modifier wrapper-style', async ({ runInlineTest }) => {
+    test.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/15239' });
+    test.fixme(true, 'Need to determine what we expect behavior here to be.');
+
+    const result = await runInlineTest({
+      'a.test.ts': `
+        const { test } = pwt;
+
+        test.describe.skip('suite1', () => {
+          test('no marker', () => {});
+          test.skip('skip wrap', () => {});
+          test('skip inner', () => { test.skip(); });
+          test.fixme('fixme wrap', () => {});
+          test('fixme inner', () => { test.fixme(); });
+        });
+
+        test('example', () => {});
+      `,
+    });
+
+
+    expect(result.exitCode).toBe(0);
+    expect(result.passed).toBe(1);
+    expect(result.skipped).toBe(5);
+
+    const expectTest = expectTestHelper(result);
+    expectTest('skip wrap', 'skipped', 'skipped', 'fixme,skip');
+    expectTest('skip inner', 'skipped', 'skipped', 'fixme,skip');
+    expectTest('fixme wrap', 'skipped', 'skipped', 'fixme,skip');
+    expectTest('fixme inner', 'skipped', 'skipped', 'fixme,skip');
```

Comment:
Let's review what expect each of these cases to be annotated as. Ignore what's in this assertion atm.